python/llm/example/GPU/HuggingFace/LLM/vicuna/README.md

```diff
@@ -118,9 +118,9 @@ Arguments info:
 > Please select the appropriate size of the Vicuna model based on the capabilities of your machine.
 
 #### Sample Output
-#### [lmsys/vicuna-13b-v1.3](https://huggingface.co/lmsys/vicuna-13b-v1.3)
+#### [lmsys/vicuna-13b-v1.5](https://huggingface.co/lmsys/vicuna-13b-v1.5)
 ```log
-Inference time: xxxx s
+Inference time: 1.0269405841827393 s
```

Comment:
Don't show real performance data in our example, just keep `Inference time: xxxx s`.